apps/web/sentry.client.config.ts

```diff
@@ -28,3 +29,7 @@ Sentry.init({
 		})
 	]
 });
+// if (process.env.NODE_ENV === 'production' && process.env.SENTRY_DSN) {
```

Comment:
@NdekoCode why this code commented out!? Also I think this code using @sentry/node page which is for SERVER, but the file called sentry.client! So it's in wrong place and commented out!

Reply:
This code was there in the test version, when I wanted to check that you can only run sentry in production mode, but I remembered that DNS will only be provided when you test or when you put the application into production.
but I left it there in case you might decide otherwise, sir @evereq